packages/client/src/components/app/forms/Form.svelte

```diff
@@ -64,7 +64,7 @@
 
   // Fetches the form schema from this form's dataSource
   const fetchSchema = async dataSource => {
-    if (dataSource?.tableId && dataSource?.type !== "query") {
+    if (dataSource?.tableId && !dataSource?.type?.startsWith("query")) {
```

Comment:
Could we not be more explicit? Could we exclude the specific types, and can we use the typed ones?

Reply:
Do you have an example of specific types being used in another svelte component?